contracts/simple-counter/Cargo.toml

```diff
@@ -28,7 +27,6 @@ backtraces = ["cosmwasm-std/backtraces"]
 # use library feature to disable all instantiate/execute/query exports
 library = []
 
-
 [dependencies]
 cosmwasm-std = "1.0.0"
 cosmwasm-storage = "1.0.0"
```

Comment:
Profile-release in Cargo.toml should be moved to root

Reply:
""Cargo only looks at the profile settings in the Cargo.toml manifest at the root of the workspace. Profile settings defined in dependencies will be ignored.""
https://doc.rust-lang.org/cargo/reference/profiles.html
The `profile` defined there will be automatically ignored. It has nothing to do with this issue.